facilitator/src/intake.rs

```diff
@@ -18,6 +20,18 @@ use slog::{debug, info, o, Logger};
 use std::{collections::HashMap, iter::Iterator};
 use uuid::Uuid;
 
+#[derive(Debug, thiserror::Error)]
+pub enum IntakeError {
+    #[error(transparent)]
+    Idl(IdlError),
+    #[error("packet decryption failure for packet {0}")]
+    PacketDecryptionError(Uuid),
```

Comment:
nit: I prefer that variants on an error enum not end in `Error`. Otherwise you get `IntakeError::PacketDecryptionError` which unnecessarily repeats `Error` (stutters, as some Go guidelines put it).

Reply:
Ah thanks, forgot this one when I moved it.